sankey_dependency_wheel_plugin/README.md

```diff
@@ -0,0 +1,196 @@
+# GoodData.UI Dashboard Plugin - Sankey Dependency Wheel
+
+This plugin replaces column charts with a widget title suffix `$WIDGET_TITLE_SUFFIX` (defined in `Plugin.jsx`) for a custom [Highcharts](https://www.highcharts.com/) [sankey dependency wheel](https://www.highcharts.com/docs/chart-and-series-types/dependency-wheel).
```

Comment:
It seems it's not replacing column charts but pivot table

Reply:
good point, thanks, fixing it now…